src/content/self-hosted/administration/configuration.mdx

```diff
@@ -517,6 +517,24 @@ The role Okteto assigns to every user. If empty, Okteto will create a default ro
 clusterRole: "role name"
 ```
 
+### user extraRoleBindings
+
+Extra `RoleBindings` to be created for every user's service account. This is useful if you want to grant access to other namespace resources in the cluster.  Disabled by default.
+
+The `RoleBindings` will reference `ClusterRoles` and will be scoped to the namespace specified in the configuration. Here's an example:
+
+```yaml
+user: 
+  extraRoleBindings:
+    enabled: true
+    roleBindings:
+      namespace-name1:
```

Comment:
what happens if the namespace doesn't exist already?

Reply:
As it is now, we will fail with an `internal server error`. We are delegating the `namespaces` and `clusterRoles` management to the admin, since this is a custom feature for them.

We are also handling the `rolebindings` resources at the minimum, always deleting the old ones when the configuration changes.

Another way to handle this could be to create the namespace ourselves or ignore the `roleBinding` if the namespace doesn't exist, but both options that could lead to silent errors

Do you have another behavior in mind?